src/extension.ts

```diff
@@ -80,10 +80,10 @@ async function getTargetServer(tomcat: TomcatController, tomcatItem ?: TomcatSer
     return server;
 }
 
-async function startServer(tomcat: TomcatController, tomcatItem ?: TomcatServer): Promise<void> {
+async function startServer(tomcat: TomcatController, tomcatItem?: TomcatServer): Promise<void> {
     const server: TomcatServer = await getTargetServer(tomcat, tomcatItem, true);
     if (server) {
-        if(server.isStarted()) {
```

Comment:
In the previous version. If the server is already started, it will stop and start, so no need for the information here.

Reply:
I will suggest that we make a restart command to silently stop then start a running server.